docs/src/examples/supplemental_material/Convex.jl_intro_ISMP2015.jl

```diff
@@ -196,22 +197,14 @@ sum(x) + y[2]
 
 2 * maximum(x) + 4 * sum(y) - sqrt(y[1] + x[1]) - 7 * minimum(x[2:4])
 
-#-
-
-## not dcp compliant
+# - not dcp compliant
 log(x) + square(x)
 
-#-
-
-## $f$ is convex increasing and $g$ is convex
+# - Composition $f\circ g$ where $f$ is convex increasing and $g$ is convex
```

Comment:
add leading characters to workaround https://github.com/JuliaDocs/Documenter.jl/issues/1483